### PR TITLE
[1LP][RFR] Flake everything in travis

### PR DESCRIPTION
**Pull request by mshriver**
Update pre-commit config to apply flake to the entire repo, instead of excluding `cfme|scripts`.

Added W504 and P665 to flake exclusions via setup.cfg

Various formatting fixes around fixing the flake issues, some BZ cleanups and such.

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit a26c92becdf5801d1609170b5e3ac4b6a581426a:

`cfme/tests/containers/test_blacklisted_container_events.py`:
- :bangbang: Line 52:19: **E121** *continuation line under-indented for hanging indent*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit de75be00e274066d1db5ef71467b18519f4a9354:

`cfme/tests/containers/test_blacklisted_container_events.py`:
- :bangbang: Line 52:19: **E121** *continuation line under-indented for hanging indent*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -25,16 +25,7 @@ repos:
...
   - id: flake8
     language_version: python3.6
-    <<: *exclude
-# separately flake via system for polarion docstrings plugins
-- repo: local
-  hooks:
-    - id: system-flake8
-      name: flake8-plugins
-      description: flake8 integration_tests plugins
-      entry: python -m flake8.__main__ --select P --ignore P665
```
Comment (mshriver) [inline .pre-commit-config.yaml]: This ignore has now been added to setup.cfg

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -25,16 +25,7 @@ repos:
...
-      files: \.py$
-      language: system
+    additional_dependencies: [polarion-docstrings]
```
Comment (mshriver) [inline .pre-commit-config.yaml]: This pulls the polarion flake plugin into the pre-commit venv

File: setup.cfg [hunk 3]
```diff
@@ -13,5 +13,5 @@ entry_points = file:entry_points.txt
 exclude = cfme.tests
 
 [flake8]
-ignore = E128,E811,W503
+ignore = E128,E811,W503,W504,P665
```
Comment (mshriver) [inline setup.cfg]: Adding W504, unless we want to enforce it? right now we violate it _everywhere_

**Pull request merged**